props/prop_base.h

```diff
@@ -855,6 +855,7 @@ class PropBase : CommandParser, Looper, protected SaberBase, public ModeInterfac
     TWIST_CLOSE,
     TWIST_LEFT,
     TWIST_RIGHT,
+    UNKNOWN_GESTURE,
```

Comment:
If you add a new value here, you should add it to GetStrokeGroup

Reply:
done